server/commands.go

```diff
@@ -82,3 +85,49 @@ func ResetAll() *cobra.Command {
 
 	return resetAll
 }
+
+func ResetState() *cobra.Command {
+	// ResetStateCmd removes the database of the specified CometBFT core instance.
+	return &cobra.Command{
+		Use:     "reset-state",
+		Aliases: []string{"reset_state"},
+		Short:   "Remove all the data and WAL",
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+
+			config := server.GetServerContextFromCmd(cmd).Config
+			appdb := filepath.Join(config.DBDir(), "application.db")
+			dymintdb := filepath.Join(config.DBDir(), "dymint")
+			settlementdb := filepath.Join(config.DBDir(), "settlement")
+			snapshotsdb := filepath.Join(config.DBDir(), "snapshots")
+			if cmtos.FileExists(appdb) {
```

Comment:
can probably have one function which accepts a path and do it vs writing the same code for all 4 paths.

Reply:
done